package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vscode-java-pack",
   "displayName": "Java Extension Pack",
-  "description": "Popular extensions for Java development and more.",
+  "description": "Popular extensions for Java development that provides Java intellisense, debugging, testing, Maven support, project management and more",
```

Comment:
intellisense -> IntelliSense. It's a general term. See https://code.visualstudio.com/docs/editor/intellisense

Maven support -> Maven/Gradle support